java/test/src/main/java/test/Ice/optional/SerializableClass.java

```diff
@@ -18,6 +18,11 @@ public boolean equals(java.lang.Object obj) {
         return false;
     }
 
+    @Override
+    public int hashCode() {
+        return _v;
+    }
+
```

Comment:
Java requires that if two things are `equal`, that their `hashCode`s must also be equal.
Right above this, we override `equals`, but weren't overriding `hashCode`, breaking this contract.